Item tooltip cleanup: Diffusal to MKB (Diffusal, MKB]

Snoopy360 opened the pull request:
removed structure note from daedalus
removed or roshan condition from heart
altered infinity bottle lore
removed Unique Attack Modifier condition from mask of madness
reworked monkey king bar to new patch

Haganeko commented:
I'm going to throw a 'no' at that lore change. It's an in joke that will make no sense to a lot of people.

chrisinajar commented on game/resource/English/ability/items/tooltip_greater_crit.txt:
```diff
@@ -4,7 +4,6 @@
 "DOTA_Tooltip_Ability_item_greater_crit_2"                              "Daedalus"
 "DOTA_Tooltip_Ability_item_recipe_greater_crit_2"                       "Daedalus Recipe"
 "DOTA_Tooltip_Ability_item_greater_crit_2_Description"                  "<h1>Passive: Critical Strike</h1>Grants each attack a %crit_chance%%% chance to deal %tooltip_crit_damage%%% additional damage."
```
Should be `<h1>Passive: Critical Strike</h1>Grants each attack a %crit_chance%%% chance to deal %crit_multiplier%%% damage.`

chrisinajar commented on game/resource/English/ability/items/tooltip_infinity_bottle.txt:
```diff
@@ -3,5 +3,6 @@
 "DOTA_Tooltip_Ability_item_infinite_bottle"                                         "Infinity Bottle"
 "DOTA_Tooltip_Ability_item_infinite_bottle_Description"                             "<h1>Active: Regenerate</h1>Consumes a charge to restore %health_restore% health and %mana_restore% mana over %restore_time% seconds.<br><br>Initial Charges: 3<br>Max Charges: Infinite"
-"DOTA_Tooltip_Ability_item_infinite_bottle_Lore"                                    "A bottle blown from the sands of time itself."
+"DOTA_Tooltip_Ability_item_infinite_bottle_Lore"                                    " the Darklord came and told the heroes. bottles now stack! thus the infintiy bottle was born"
```
cool meme but pretty bad writing

chrisinajar commented on game/resource/English/ability/items/tooltip_mask_of_madness.txt:
```diff
@@ -3,7 +3,7 @@
 "DOTA_Tooltip_Ability_item_mask_of_madness_2"               "Mask of Madness"
 "DOTA_Tooltip_Ability_item_recipe_mask_of_madness_2"        "Mask of Madness Recipe"
-"DOTA_Tooltip_Ability_item_mask_of_madness_2_Description"                           "<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack d ...
+"DOTA_Tooltip_Ability_item_mask_of_madness_2_Description"                           "<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack d ...
```
There's an extra space before "Lasts". It should be `<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack damage dealt.`

Snoopy360 commented:
changed spell damage to magic damage in pipe
removed UAM from satanic description
removed "reduce damage output from silver edge
reset urn description to dota2 vanila 
added note to spirit vessel about disabling on enemy hero damage.
Extra things I noticed
hurricane pike is doesn't mention ranged only effect
lotus orb doesn't mention soft purge ability
meteor hammer not really implimented yet
error in game with urn of shadows. the duration isn't displaying.

Snoopy360 commented:
all changes requested have been made tooltips = done (hopefully)

imaGecko commented:
Requested changes from the other now closed pull requests seem to be in

chrisinajar commented:
I'm going to close this PR since we totally changed the way tooltips are maintained. Feel free to update your branch and re-open the PR, or do the work in a new branch. Both work for me.
